client/client.go

```diff
@@ -247,6 +247,12 @@ func (cli *Client) UpdateClientVersion(v string) {
 
 }
 
+// ClientHost returns the host associated with this instance of the Client.
+// This operation doesn't acquire a mutex.
+func (cli *Client) ClientHost() string {
```

Comment:
Yes, I had my doubts about the name as well, but couldn't come up with a good one. `Host` sounds good to me (or `DaemonHost`) :D